### PR TITLE
Annotate new transcriptome using existing guide annotations

### PR DESCRIPTION
**Pull request by kvg**

### CODE REVIEW

File: wdl/AnnotateTranscriptomeWithGuide.wdl [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+            gtf = QuantifyWithoutRetainedIntrons.st_gtf
+    }
+
+    call StringTie2.ExtractTranscriptSequences as ExtractTranscriptsWithRetainedIntrons {
+        input:
+            ref_fasta = ref_map['fasta'],
+            ref_fasta_fai = ref_map['fai'],
+            gtf = QuantifyWithRetainedIntrons.st_gtf
+    }
+
```
Comment (jonn-smith) [inline wdl/AnnotateTranscriptomeWithGuide.wdl:66]: Can you add in a step to create a sequence dictionary here as well with Picard / GATK `CreateSequenceDictionary`?  Most of the time having the dictionary around is helpful.

**Review by jonn-smith — CHANGES_REQUESTED** [review]
A couple of straight-forward suggested code changes and requests.

File: wdl/AnnotateTranscriptomeWithGuide.wdl (hunk 1 above)
Comment (kvg) [reply wdl/AnnotateTranscriptomeWithGuide.wdl:66]: Will do - I imagine CreateSequenceDictionary or `samtools dict` makes no difference to you?

Comment (jonn-smith) [reply wdl/AnnotateTranscriptomeWithGuide.wdl:66]: Yup!  Either should work just fine.

Comment (kvg) [reply wdl/AnnotateTranscriptomeWithGuide.wdl:62]: Done!

**Review by jonn-smith — APPROVED** [review]
👍

**Pull request merged**